app/Exceptions/Handler.php

```diff
@@ -21,11 +21,7 @@ class Handler extends ExceptionHandler
      *
      * @var array<int, string>
      */
-    protected $dontFlash = [
-        'current_password',
-        'password',
-        'password_confirmation',
-    ];
+    protected $dontFlash = ['current_password', 'password', 'password_confirmation'];
```

Comment:
I'm so conflicted by these original multi-line to single line changes 😬 

Can we keep them multiline by adding a comment on the first entry and avoid this weird formatting?

Reply:
It's formatted this way because Laravel is not using Prettier, and we are probably not using what Laravel is using. I don't think we should waste time discussing Prettier opinionated decisions and keep trying to circumvent these things, because someone will break this (remove whatever is preventing the line to be single) on their codebase and Prettier will probably make it a single line o a new codebase. So why not just keep formatting the way Prettier will do anyway?